.circleci/config.yml

```diff
@@ -436,6 +436,11 @@ jobs:
           use_oidc: true
           workload_identity_pool_id: GCP_OIDC_WIP_ID
           workload_identity_pool_provider_id: GCP_OIDC_WIP_PROVIDER_ID
+      - attach_workspace:
+          at: /tmp/cache
+      - run:
+          name: Restore Docker image cache
+          command: docker load -i /tmp/cache/docker.tar
```

Comment:
Just a quick question: is there any concern of caching-related issues with docker image artifacts? I recall some issues in Merino when we'd make changes where there would be an out of date docker image that would break things in CI.  Not super specific, but curious of any possibilities of issues, since you know more about this than I do!